tests/unit/utils/test_errors_parsing.py

```diff
@@ -6,46 +6,49 @@
 
 from ols.utils import errors_parsing
 
+BAD_REQUEST_STATUS_CODE = 400
+
 
 def test_parse_generic_llm_error_on_bad_request_error_without_info():
     """Test the parse_generic_llm_error function when BadRequestError is passed."""
-    status_code = 400
-    response = Response(status_code=status_code, request={})
+    expected_status_code = BAD_REQUEST_STATUS_CODE
```

Comment:
Do we need to create a new variable to give it the exact same value as the global one?

Reply:
it's just to have it named